spec/jobs/clean_up_user_export_topics_spec.rb

```diff
@@ -5,6 +5,11 @@
 RSpec.describe Jobs::CleanUpUserExportTopics do
   fab!(:user) { Fabricate(:user) }
 
+  after do
+    I18n.locale = :en
+    I18n.reload!
+  end
```

Comment:
In theory this shouldn't be required because we reset the locale before every test 🤔 

https://github.com/discourse/discourse/blob/178b294a629cb7fa6970c45be07c71d5305910a9/spec/rails_helper.rb#L126-L126

Do you know why the general fix wasn't working here?

Reply:
Good point! I dug into it and opened #13389 instead. 🙂